### ISSUE TITLE
Add accept terms and conditions pop-up to Vortex

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/vortex#103]
## Context 
As we would be collecting user interaction data using google analytics and also capturing the email address of the data, this ticket is to add a pop-up to the vortex app so the user can accept the terms and conditions before interacting with the app

## Requirement
* Show a pop-up before user interacts with the Vortex app
* The pop-up design and copy should be replicated from what we have on Portal

## Notes
* Product will add a link to the terms and conditions for Vortex

## Reference
<img width="580" alt="Screenshot 2024-08-14 at 6 03 21 PM" src="https://github.com/user-attachments/assets/79c6991c-0306-4c7c-a6dd-80b965712bfb">

**Comment by TorstenStueber** [comment]
The website https://www.vortexfinance.co/legal/terms-of-service does not exist yet. So this is still on hold, not?

**Comment by prayagd** [comment]
Now unblocked the terms page is added https://www.vortexfinance.co/terms-conditions @Sharqiewicz can you please link it?
